extension/src/popup/views/ManageCollectibles/index.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from "react";
+import { Formik, Form, Field, FieldProps } from "formik";
+import { useTranslation } from "react-i18next";
+import { Button, Input, Icon } from "@stellar/design-system";
+import { object as YupObject, string as YupString } from "yup";
+import { useNavigate } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { captureException } from "@sentry/browser";
+
+import { publicKeySelector } from "popup/ducks/accountServices";
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { Loading } from "popup/components/Loading";
+import { SubviewHeader } from "popup/components/SubviewHeader";
+import { View } from "popup/basics/layout/View";
+import { FormRows } from "popup/basics/Forms";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import { isContractId } from "@shared/api/helpers/soroban";
+import { addCollectible } from "@shared/api/internal";
+import { useGetCollectibles } from "helpers/hooks/useGetCollectibles";
+import {
+  RequestState,
+  useGetManageCollectiblesData,
+} from "./hooks/useGetManageCollectiblesData";
+
+import "./styles.scss";
+
+import { TabsList } from "../Account/contexts/activeTabContext";
+
+interface FormValues {
+  collectibleAddress: string;
+  collectibleTokenId: string;
+}
+const initialValues: FormValues = {
+  collectibleAddress: "",
+  collectibleTokenId: "",
+};
+
+export const ManageCollectibles = () => {
```

Comment:
how do we feel about renaming this component to `AddCollectible`? I think that would be a little more assertive since that's the only functionality this component has - Wdyt?